tests/integration/examples/01_create_reqif_objects/script.py

```diff
@@ -63,15 +63,47 @@ class ExampleIdentifiers:
     attribute_definitions=[requirement_text_attribute],
 )
 
-spec_object_attribute = SpecObjectAttribute(
-    attribute_type=SpecObjectAttributeType.STRING,
-    definition_ref=ExampleIdentifiers.SPEC_ATTRIBUTE_ID,
-    value="System ABC shall do XYQ.",
-)
+spec_object_attributes = [
+    SpecObjectAttribute(
+        attribute_type=SpecObjectAttributeType.STRING,
+        definition_ref=ExampleIdentifiers.SPEC_ATTRIBUTE_IDs[0],
+        value="System ABC shall do XYQ.",
+    ),
```

Comment:
Thanks for showing it with a test. This could be a useful change for the use case that you described.

Let's also add a validation inside the init method: if an `int` value is provided than the type should be INTEGER and similar for all other types. If these is a type mismatch, let's through a TypeError.